app/src/main/java/aniyomi/source/SourcesIds.kt

```diff
@@ -0,0 +1,5 @@
+package aniyomi.source
```

Comment:
Is this even used anywhere?
Sorry if I missed something

Reply:
Okay but I don't see how this has any relevance because I believe this is related to SY's "special" sources

Reply:
@Quickdesh I think you used SY, right? What is this for?

Reply:
Merged Manga and Lewed are SY special sources, yes. These can be removed since they don't affect Aniyomi